### PR TITLE
Docs contributions

### PR DESCRIPTION
**Pull request by benpowis**
Hi all,

I'd like to contribute to the docs with a blog post that I've written. Where would be the best place to put this? Is it okay to contribute markdown rather than adding this to the existing html framework?

I will customize the content to give it better context as part of the docs but wanted to be clear on these points first.

Ben

### CODE REVIEW

**Comment by benpowis** [comment]
I signed it!

**Comment by garanj** [comment]
Ben, looks good! Thanks for doing this!

To answer questions about where this should live and use of markdown. Alex and Pierre may jump in, but I think this lives in the Quickstart section here: https://google.github.io/crmint/docs/ my reasoning being that it reads as a whistlestop tour for someone who wants to know how to do all the main actions in CRMint. 

It also ties in nicely with the deployment quickstart which can be linked off your existing text.

If it were to live there, then there would be under Quickstart:

- Try deploying CRMint
- [[This intro]]
- Run a BigQuery ML pipeline (coming soon - Pierre is just merging this!)

Which feels a good mix, from the basic, through to the more in-depth

The documentation is actually built from markdown, and lives in `docsource/`. The reason it is built is that the worker reference is built from the worker source code using a custom Ruby gem that isn't part of the GitHub Jekyll build.

You can find the build instructions in the `docsource/` directory.

I'd suggest adding this to the `docsource/docs/quickstart` directory. You'll also want to add it to the menus e.g. files such as `docsource/_includes/doc-side-quickstart-nav.html` which gets pulled in in the build.

Naming-wise maybe *a whistlestop tour of CRMint* or something?

Other than that, I'll defer to what Alex and Pierre suggest!

File: docsource/_includes/doc-side-quickstart-nav.html [hunk 1]
```diff
@@ -2,4 +2,5 @@
 	{% assign current = page.url | downcase | split: '/' %}
 	<li><h5 class="doc-side-nav-title">Quick Start</h5></li>
 	<li class="doc-side-nav-list-item"><a href="{{ site.baseurl }}/docs/quickstart/" {% if !current[3] %}class='current'{% endif %}>Try deploying CRMint</a></li>
+	<li class="doc-side-nav-list-item"><a href="{{ site.baseurl }}/docs/quickstart/Introduction.md" {% if !current[3] %}class='current'{% endif %}>A whistlestop tour of CRMint</a></li>
```
Comment (garanj) [inline docsource/_includes/doc-side-quickstart-nav.html]: Would it be ok to change this to a lowercase introduction not Introduction, just for consistency.

File: docs/Introduction.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
+### Data pipelines for everyone with CRMint
+#### Automate your BigQuery SQL queries with this elegant GCP pipeline application.
+
+![](https://cdn-images-1.medium.com/max/800/1*QSxSRgKOzt39wrY19Fa1hg.jpeg)
```
Comment (garanj) [inline docs/Introduction.md]: Could we perhaps move the images to our own documentation, in preference to relying on medium.com?

Perhaps under `docsource/img/`

File: docs/Introduction.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+#### Inline functions
+
+As well as custom variables, there are a handful of functions built into the core of CRMint that you may find helpful.
+
+For example **{% today(‘%Y-%m-%d’) %}** will insert today’s date in the YYYY-MM-DD format. You can use these functions in your query or in any of the destination fields; great if you want your query to write out to a different table every day, or perhaps to a .csv file on Google Cloud Storage with a unique name every day.
+
+Here are a couple more example of inline functions in action:
+
+-   **my_file_{% day_ago(3, ‘%Y-%m-%d’) %}.csv
```
Comment (garanj) [inline docs/Introduction.md]: Something doesn't render quite right here. Not sure what...?

**Comment by dulacp** [comment]
Hi Ben,

I realize that we never closed/merged these changes. Were you able (back then) to implement the changes suggested? 

Closing it for now, but I can re-open this PR in case you reach out to us.

Cheers

**Pull request closed without merging**